components/feature/downloads/src/main/res/layout/mozac_downloads_prompt.xml

```diff
@@ -64,12 +64,13 @@
         android:layout_alignBaseline="@id/icon"
         android:layout_marginStart="3dp"
         android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
         android:layout_toEndOf="@id/icon"
         android:paddingStart="5dp"
         android:paddingTop="4dp"
         android:paddingEnd="5dp"
         android:textColor="?android:attr/textColorPrimary"
-        tools:text="Firefox_Preview_v2.1.apk"
+        tools:text="AnCmIdj4Z65FxG3vOmO6sbbEsG74MNpWM5EVwnQVKMgQP5bAVQ7ryKLCJ1Qd5XD3894rSoLmZYPBtcTy37XtohEsJeBZMmURUr2WPCTC4IjQq7yy8KKkiweXmerjYp1Onlkt3M07ukAsJ2oMRkfX3GhQuc0t00hDVSAOvGxn4qfuoT3AWP02SwfG9VjvoWa8Rv7BmRHofc4RpCiMi5H0pbFYttHab5GGw04Qbh56QGGlAFJGmnKj3sF58Jt.apk"
```

Comment:
You could also use the built-in lorem ipsum generator if you need garbled text to fill up a TextView.

```suggestion
        tools:text="@tools:sample/lorem/random"
```

Reply:
thanks for the advice! added to the commit manually